code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -60,12 +67,33 @@
 				to_chat(user, "<span class='danger'>You shouldn't be reading this message! Contact a coder or someone, something broke!</span>")
 				IED = null
 				return
+	if(istype(I, /obj/item/device/assembly/signaler))
+		if(IED)
+			to_chat(user, "<span class='warning'>This beartrap already has an IED hooked up to it!</span>")
+			return
+		if(sig)
+			to_chat(user, "<span class='warning'>This beartrap already has a signaler hooked up to it!</span>")
+			return
+		sig = I
+		if(sig.secured)
+			to_chat(user, "<span class='notice'>The signaler is secured.</span>")
+			sig = null
+			return
+		user.drop_item(src)
```

Comment:
[Read the aggressive comment](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/inventory.dm#L91).